dev-util/intel-graphics-compiler/intel-graphics-compiler-1.0.9636.ebuild

```diff
@@ -0,0 +1,112 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+CMAKE_BUILD_TYPE="Release"
```

Comment:
Not needed with EAPI8 then.

Reply:
Because is based on intel-graphics-compiler-1.0.9441.ebuild  ??